UE/Tests/Application/Ports/UserPortTestSuite.cpp

```diff
@@ -54,7 +57,33 @@ TEST_F(UserPortTestSuite, shallShowMenuOnConnected)
     EXPECT_CALL(guiMock, setListViewMode()).WillOnce(ReturnRef(listViewModeMock));
     EXPECT_CALL(listViewModeMock, clearSelectionList());
     EXPECT_CALL(listViewModeMock, addSelectionListItem(_, _)).Times(AtLeast(1));
+    EXPECT_CALL(guiMock, setAcceptCallback(_));
     objectUnderTest.showConnected();
 }
 
+TEST_F(UserPortTestSuite, shallViewSmsList)
+{
+    using SmsMessages = std::vector<std::pair<Sms, SmsState>>;
+    const SmsMessages smsMessages;
+
+    EXPECT_CALL(guiMock, setListViewMode()).WillOnce(ReturnRef(listViewModeMock));
+    EXPECT_CALL(listViewModeMock, clearSelectionList());
+    EXPECT_CALL(listViewModeMock, addSelectionListItem(_, _)).Times(AtLeast(0));
```

Comment:
at least 0...

Reply:
Because, sms messages list might be empty.

Reply:
so what changes if you remove this line

Reply:
I think, unexpected call error might appear. listViewModeMock is strickMock

Reply:
that's right, my bad. good job